lib/features/title_header.dart

```diff
@@ -9,41 +9,45 @@ class TitleHeader extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Padding(
-      padding: const EdgeInsets.fromLTRB(10, 10, 10, 10),
+      padding: EdgeInsets.all(
+        MediaQuery.of(context).size.width > 380 ? 10 : 5,
+      ),
       child: Row(
         mainAxisAlignment: MainAxisAlignment.spaceBetween,
         crossAxisAlignment: CrossAxisAlignment.center,
         children: [
-          Image.asset(
-            'assets/logo.png',
-            height: 100,
-          ),
-          Column(
-            mainAxisAlignment: MainAxisAlignment.center,
-            crossAxisAlignment: CrossAxisAlignment.start,
+          Row(
+            mainAxisSize: MainAxisSize.min,
             children: [
+              Image.asset(
+                'assets/logo.png',
+                height: MediaQuery.of(context).size.width > 380 ? 100 : 70,
+              ),
+              SizedBox(
+                width: MediaQuery.of(context).size.width > 380 ? 10 : 5,
```

Comment:
what is 380 ?

Reply:
It's a Breakpoint under which there is an overflow